proposed/2022/PMUIPackageSourceMapping.md

```diff
@@ -0,0 +1,72 @@
+# PMUI 
+* Start Date: 6/25/22
+* Authors: Ella McNally
+* Issue
+
+## Summary
+
+Currently there is no support for package source mapping in the PMUI. I will add a feature that allows the user to choose a source to map a package to when they install it and a feature that shows all package source mappings in the preview. This feature is a portion of what was discussed in a previous spec. https://github.com/NuGet/Home/blob/dev/implemented/2021/PackageSourceMapping.md
```

Comment:
Great summary. Perhaps two paragraphs to separate each problem being solved here.

1. Mapping a source when installing a package.
2. Managing mapped sources in the options.